File: src/main/java/com/agashchuk/vueDemo/model/Role.java
package com.agashchuk.vueDemo.model;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table(name = "ROLE")
@Data
public class Role implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "ID")
    private Long id;

    private String name;

    public String getAuthority() {
        return name;
    }

    public void setAuthority(String authority) {
        this.name = authority;
    }

}

File: src/main/kotlin/com/agashchuk/vueDemo/model/Role.kt
package com.agashchuk.vueDemo.model

import javax.persistence.*

@Entity
@Table(name = "ROLE")
data class Role(@Id
                @GeneratedValue(strategy = GenerationType.AUTO)
                @Column(name = "ID") val id: Long?,
                val name: String? = null) {
    constructor() : this(id = null)
}